packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
+import { expect } from '@playwright/test';
+import { dataUiId, stableScreenshot, waitForSelector } from '../../common/utils';
+import { IDS } from '../../common/constants';
+
+test.describe('Rooms CallScreen tests for different roles', async () => {
+  test('All CallControls are enabled for Presenter', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Presenter' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot('rooms-call-screen-presenter.png');
+  });
+
+  test('All CallControls are enabled for Attendee', async ({ page, serverUrl }) => {
```

Comment:
Screenshare should not be enabled for an attendee (correct in the snapshots but this test text is incorrect)